ethcore/types/src/client_types.rs

```diff
@@ -67,10 +66,10 @@ pub struct ClientReport {
 
 impl ClientReport {
 	/// Alter internal reporting to reflect the additional `block` has been processed.
-	pub fn accrue_block(&mut self, header: &Header, transactions: usize) {
+	pub fn accrue_block(&mut self, gas_used: &U256, transactions: usize) {
 		self.blocks_imported += 1;
 		self.transactions_applied += transactions;
-		self.gas_processed = self.gas_processed + *header.gas_used();
+		self.gas_processed = self.gas_processed + gas_used;
```

Comment:
```suggestion
		self.gas_processed += gas_used;
```

Reply:
I don't think that works on `U256`, but I might be wrong?

Reply:
It works, but needs a deref.

Reply:
maybe better to pass it by value then? deref does a copy